cabal.project

```diff
@@ -119,8 +119,8 @@ source-repository-package
 source-repository-package
     type: git
     location: https://github.com/input-output-hk/cardano-addresses
-    tag: b7273a5d3c21f1a003595ebf1e1f79c28cd72513
-    --sha256: 129r5kyiw10n2021bkdvnr270aiiwyq58h472d151ph0r7wpslgp
+    tag: 3172d30433b4cdd9beabe6b5a0a887351f5c0061
```

Comment:
I can't find this commit in the repo yet? 🤔 

https://github.com/input-output-hk/cardano-addresses/pull/202/commits/3172d30433b4cdd9beabe6b5a0a887351f5c0061